lib/commandlineio.rb

```diff
@@ -0,0 +1,48 @@
+require_relative "blackjack_score"
+
+class CommandlineIO
+
+  def initialize(output = $stdout, input = $stdin)
+    @output = output
+    @input = input
+  end
+
+  def ask_how_many_players
+    @output.puts "1 or 2 player?"
+  end
+
+  def get_num_of_players
+    num_of_players = @input.gets.chomp.to_i
+    until num_of_players == 1 || num_of_players == 2
+      @output.puts "Please enter 1 or 2"
+      num_of_players = @input.gets.chomp.to_i
+    end
+    num_of_players
+  end
+
+  def display_initial_hand(initial_hand)
+		@output.print initial_hand
+	end
```

Comment:
My spacing of def display_initial_hand on Atom looks ok. Unless, you mean the spacing on def get_num_of_players?